### DISCUSSION TITLE
New bootstrap/runtime for 0.8-RC?

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/WindowsAppSDK#947] [Q&A] [answered]
Is there a new bootstrap/runtime for 0.8-RC? Or is `https://aka.ms/projectreunion/0.8preview` still compatible? Seems to be fine, but just curious if that's supported. Thanks!

**Comment by DrusTheAxe** [comment, marked as answer]
Sorry Rafael but the Bootstrap API can't be used be used with 0.8-Rc.

0.8-preview is effectively the experimental channel (but created before the channel lingo was adopted). It has the Bootstrap API, Dynamic Dependencies and supports unpackaged apps.

0.8-RC is effectively the preview channel (but created before the channel lingo was adopted). That lacks the Boostrap API, Dynamic Dependencies and has no support for unpackaged apps.

The Bootstrap API relies on more than just Microsoft.ProjectReunion.Bootstrap.dll e.g. internally it uses Dynamic Dependencies (and very carefully). Depending on what you've got installed the `MddBootstrapInitialize()` may not fail but if so it's a lucky (and unsupported) coincidence.

**Reply by riverar** [reply]
Thanks for following up on this. Makes much more sense now with all that channel planning (and Mdd discussion in other issues) behind us. Onward!
